python/lsst/ts/cbp/mock_server.py

```diff
@@ -215,7 +216,7 @@ async def do_new_azimuth(self, azimuth):
         str
         """
         self.set_constrained_position(float(azimuth), self.encoders.azimuth)
-        return ""
+        return ":"
```

Comment:
Please extract `":"` to a constant and use that here and in all relevant lines below as well.